confluent_server/confluent/xcclient/attributes.py

```diff
@@ -0,0 +1,93 @@
+###############################################################################
+# IBM(c) 2019 EPL license http://www.eclipse.org/legal/epl-v10.html
+###############################################################################
+# -*- encoding: utf-8 -*-
+
+
+# 'node', which can be considered a 'system' or a 'vm'
+nodeattrs = {
+    'groups': 'nodelist.groups',
+    'collective.manager': 'nodehm.mgt',
+    'discovery.passwordrules': None,
+    'discovery.policy': ['switch', 'mtms', 'sequential'],
+    'info.note': 'nodelist.comments',
+    'location.room': 'nodepos.room',
+    'location.row': 'nodepos.comments', 
+    'location.rack': 'nodepos.rack',
+    'location.u': 'nodepos.u',
+    'console.logging': 'site.consoleondemand',
```

Comment:
While the intent is the same, the values are different, e.g. the console.logging values are 'full', 'interactive', and 'none'.